sdk/avalon_sdk/work_order/work_order_params.py

```diff
@@ -98,20 +96,27 @@ def set_workload_id(self, workload_id):
 
     def set_requester_id(self, requester_id):
         """Set requesterId work order parameter."""
+        if requester_id is None or \
+                not is_valid_hex_str(requester_id):
+            raise ValueError("Empty or Invalid data \
+format for requester_id")
```

Comment:
Use this form instead (similar to C++) to continue strings:

```
raise ValueError("Empty or Invalid data "
    "format for requester_id")
```

Reply:
done